compiler/src/main/resources/play_scala_security_extractors.mustache

```diff
@@ -8,6 +8,11 @@ import de.zalando.play.controllers.SwaggerSecurityExtractors._
 import {{import.name}}
 {{/for}}
 
+object SecurityExtractorsExecutionContext {
+    // TODO override with proper ExecutionContext instance
+    implicit val ec = scala.concurrent.ExecutionContext.global
```

Comment:
Yep, exactly how I'm doing that :) Scala one is bad because if it's overused potentially every Future will starve.It's also not well-suited for blocking and I/O operations because it's configuration are CPU-based.